authors/settings.py

```diff
@@ -19,7 +20,7 @@
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3'
+SECRET_KEY = getenv('SECRET_KEY')
 
```

Comment:
This a good idea to use environment variables. Although for this to work you may need to specify where the .env file will be found, look at [this doc](http://django-environ.readthedocs.io/en/latest/)